test/manifests/crd/schema/appproject-crd.yaml

```diff
@@ -0,0 +1,326 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
If this file is copied from argoproj/argo-cd, it would be good to have a makefile command that does the copy to update it.

Reply:
This is mainly for testing and even if Argo CD spec changes, it wouldn't be an issue as the main purpose is to validate that this controller won't touch AppProject fields beside the .spec.roles.